moxie/cores/ssh.py

```diff
@@ -76,7 +76,7 @@ def exit(stdin, stdout, stderr, args=None):
 def readl(stdin, stdout, echo=True):
     buf = ""
     while not stdin.at_eof():
-        bytes_ = (yield from stdin.read())
+        bytes_ = (yield from stdin.read(1))
```

Comment:
This is going to be super thrashy, it's going to result in a lot of coroutine jitter - I'd at least read a larger hunk, IIRC, read() will shot-read, that's just the max size of the read